dev/release/02-source.sh

```diff
@@ -97,16 +113,22 @@ ${SOURCE_DIR}/run-rat.sh ${tarball}
 
 # sign the archive
 gpg --armor --output ${tarball}.asc --detach-sig ${tarball}
-sha1sum $tarball > ${tarball}.sha1
-sha256sum $tarball > ${tarball}.sha256
-sha512sum $tarball > ${tarball}.sha512
+shasum -a 1 $tarball > ${tarball}.sha1
+shasum -a 256 $tarball > ${tarball}.sha256
 
 # check out the arrow RC folder
 svn co --depth=empty https://dist.apache.org/repos/dist/dev/arrow tmp
 
 # add the release candidate for the tag
-mkdir -p tmp/${tagrc}
+mkdir -p tmp/${tagrc}/binaries
+
+# copy the rc tarball into the tmp dir
 cp ${tarball}* tmp/${tagrc}
+
+# copy binary artifacts into a subdirectory of the rc dir
+cp -rf "$artifact_dir"/* tmp/${tagrc}/binaries/*
```

Comment:
i think that `*` in `tmp/${tagrc}/binaries/*` is needless. It may cause a "glob not match" error.

Reply:
Ah good point. Thank you.